tests/foreman/ui/test_user.py

```diff
@@ -23,7 +23,7 @@
 from robottelo.ui.session import Session
 
 
-def valid_strings(len1=100):
+def valid_strings(len1=10):
```

Comment:
agree with @sghai, using long chars is leading to the string being displayed as "test122...." instead of the entire length "test1223455".